add the instruction to install cluster local gateway (#1911)
knative/docs: docs/install/installing-istio.md

```

> ### Installing `cluster-local-gateway` for internal cluster-only routes

Reviewer: I editing this section for clarity and organization. PTAL and ensure its accuracy.

You can configure your Knative cluster to use an 
[Istio gateway](https://istio.io/docs/reference/config/networking/v1alpha3/gateway/) that routes 
all traffic internally. To be able to configure your revisions to use routes that are visible only within your cluster, 

> you install and use the `cluster-local-gateway` gateway.

Author: I left a comment above which also works for this part.

kubectl apply -f istio-local-gateway.yaml
```

> Alternatively, if you want to install cluster local gateway for **development purpose**, run below command 

Author: @samodell I accidentally lost Richie's  suggestion. Could you let me know how the change in this file looks?

> Alternatively, if you want to install cluster local gateway for **development purpose**, run below command 

Reviewer 2: ```suggestion
Alternatively, if you want to install the cluster local gateway for **development purposes**, enter the following command 
```

without `helm` for an easy installation:

```shell
# Istio minor version should be 1.2 or 1.3
export ISTIO_MINOR_VERSION=1.2

export VERSION=$(curl https://raw.githubusercontent.com/knative/serving/master/third_party/istio-${ISTIO_MINOR_VERSION}-latest)

kubectl apply -f https://raw.githubusercontent.com/knative/serving/master/third_party/${VERSION}/istio-knative-extras.yaml
```

> **Note:** Above installation method is only for development purpose. The production readiness of above 

Reviewer 2: ```suggestion
**Note:** This method is only for development purposes. The production readiness of the above 
```

> installation method is not ensured. For production installation, refer the `helm` installation method in this

Reviewer 2: ```suggestion
installation method is not ensured. For a production-ready installation, see the `helm` installation method above.
```

> section.

Reviewer 2: ```suggestion
```
